src/runtime/thread_pool.cc

```diff
@@ -370,11 +374,7 @@ class ThreadPool {
   // number of workers used (can be restricted with affinity pref)
   int num_workers_used_;
   // if excluding worker 0 and using master to run task 0
-#ifndef _LIBCPP_SGX_CONFIG
-  bool exclude_worker0_{true};
```

Comment:
shall we keep default values for SGX unchanged?

Reply:
This doesn't change the default value of SGX. Here is if not define, i.e. normal backend.

Reply:
My mistake, thanks